probe-rs/src/architecture/arm/sequences/nrf53.rs

```diff
@@ -50,7 +51,8 @@ impl Nrf for Nrf5340 {
     ) -> Result<bool, crate::Error> {
         let csw: CSW = arm_interface
             .read_raw_ap_register(ahb_ap_address, 0x00)?
-            .into();
+            .try_into()
+            .map_err(|e| DebugProbeError::from(e))?;
```

Comment:
This `map_err` seems redundant.

Reply:
No, it's necessary because the error cannot be converted directly.